packages/paste-website/src/pages/components/tooltip/index.mdx

```diff
@@ -0,0 +1,222 @@
+import Changelog from '@twilio-paste/tooltip/CHANGELOG.md';
+import {Tooltip} from '@twilio-paste/tooltip';
+import {Callout, CalloutTitle, CalloutText} from '../../../components/callout';
+import {SidebarCategoryRoutes} from '../../../constants';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/tooltip"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/tooltip/"}}) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name={props.pageContext.frontmatter.title}
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/master/packages/paste-core/components/tooltip"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/components-tooltip--default"
+  data={props.data.allPasteComponent.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<TableOfContents headings={props.data.mdx.headings} />
+
+<content>
+
+<Callout>
+  <CalloutTitle>Using alpha components in your product</CalloutTitle>
+  <CalloutText>
+    Alpha components are ready for you to use in production. Components such as Tooltip are likely to not see any API
+    changes. This is only an alpha due to the lack of documentation and a matching Sketch symbol. Feel free to share
+    your experience using this component by{' '}
+    <Anchor href="https://github.com/twilio-labs/paste/discussions/new">starting a discussion on GitHub</Anchor>. We are
+    also looking for a contributor to symbolize this component in Sketch. Interested? Reach out!
+  </CalloutText>
+</Callout>
+
+## Guidelines
+
+### About Tooltip
+
+The Tooltip component is a popup that displays text related to an element when the element is focused via
+the keyboard or on mouse hover. Tooltip follows the
+[WAI-ARIA Tooltip Pattern](https://www.w3.org/TR/wai-aria-practices/#tooltip).
+
+### Accessibility
+
+Tooltips follow these accessibility guidelines:
+
+- Tooltip has the role `tooltip`
+- The child element has `aria-describedby` referring to the Tooltip
+- The child element **must be** focusable
+- Pressing the escape key hides the current visible tooltip.
+
+## Examples
+
+### Basic Tooltip
+
+<LivePreview
+  scope={{
+    Tooltip,
+    Button,
+  }}
+>
+  {`<Tooltip baseId="tooltip" text="Black Lives Matter. We stand with the Black community.">
+  <Button variant="primary">✊ Action</Button>
+</Tooltip>`}
+</LivePreview>
+
+### Controlling placement
+
+The placement of the tooltip is configurable via the `placement` prop. The available placement options are
+available in the [props description here](#tooltip).
+
+<LivePreview
+  scope={{
+    Tooltip,
+    Button,
+  }}
+>
+  {`<Tooltip baseId="tooltip-right" text="Welcome to Paste!" placement="right-start">
+  <Button variant="primary">Open Tooltip</Button>
+</Tooltip>`}
+</LivePreview>
+
+### Custom element
```

Comment:
Unclear what custom means here. Would it be better to say "Passing tooltips on typically unfocusable elements"? 

And if so, we should provide guidance saying that passing tooltips on items that users may not be able to guess as tooltips is probably not a good idea

Reply:
I've changed this title, and added some more descriptive text, and a call out about the focusable element / abuse. Let me know if that clears things up.